plugins/wls.yaml

```diff
@@ -0,0 +1,18 @@
+apiVersion: krew.googlecontainertools.github.com/v1alpha2
+kind: Plugin
+metadata:
+    name: wls
+spec:
+    version: v1.0.0
+    homepage: 'https://github.com/mohatb/kubectl-wls'
+    shortDescription: 'Windows-linux-shell allows you to start a shell on windows or linux nodes'
+    description: "kubectl-wls stands for kubectl Kubectl-Windows-Linux-Shell, it is an interactive shell script and also a kubectl plugin that allows you to access a kubernetes windows or linux nodes shell.\n"
```

Comment:
First, please refer to other plugin manifest to see how to write multi-line strings that are wrapped at 80 chars.

Then, I don't understand what the plugin does from this sentence either.

> kubectl-wls stands for kubectl Kubectl-Windows-Linux-Shell

not useful

> it is an interactive shell script and also a kubectl plugin

sounds redundant

> allows you to access a kubernetes windows or linux nodes shell

what is a "kubernetes windows"? "linux nodes"?

Please use improved grammar.

Reply:
in kubernetes, there is a windows nodes and linux nodes, this allows the user to ssh to any of those nodes. this does not need a grammer expert

Reply:
in kubernetes, there are windows or Linux nodes, this supports accessing shell in either windows or Linux nodes.